### ISSUE TITLE
Version file cannot be parsed any longer (compiled task missing)

### ISSUE THREAD

**Issue by rdeago** [Buildvana/Buildvana.Sdk#162]
## Current version

1.0.0-alpha.20, after committing #158 and #161

## Description

**TL;DR:** I messed up. Badly.

The code generation tasks are gone... and with them the `ParseVersionFile` task!

Yeah, I'm a forking genius. The need to get alpha 21 out the door ASAP made me skip serious testing. I should know better, I _do_ know better, but sometimes I let rush (and sleep deprivation) get the better of me.

## Configuration

All configurations, even when setting `UseVersionFile` to `false` (because #161 is buggy, too).

## Regression?

It's a regression with respect to 1.0.0-alpha.20.

## Other information

Version file parsing has to generate MSBuild properties, thus it just cannot be implemented with a source generator.

Bringing back the compiled tasks, for a feature I'm trying to kill, just doesn't feel right.

The best thing to do is probably to just ditch version file parsing completely, wrap up the experiments with `Nerdback.GitVersioning` and/or other versioning systems, and implement one of those instead.

**Comment by rdeago** [comment]
So be it: let's strip version file treatment from alpha 21. As breaking a change as can be. _\<sigh/\>_

Alpha 22 will have to solve the versioning nightmare once and for all, though.
